helix-term/src/commands.rs

```diff
@@ -397,6 +397,7 @@ impl MappableCommand {
         smart_tab, "Insert tab if all cursors have all whitespace to their left; otherwise, run a separate command.",
         insert_tab, "Insert tab char",
         insert_newline, "Insert newline char",
+        insert_char_interactive, "Insert an interactively-chosen char",
```

Comment:
i'm not super happy with this name or description, but hopefully it gets the idea across.